pytest-selenium/tests/test_attribution.py

```diff
@@ -49,3 +50,31 @@ def test_attribution_collapsed_by_default_expands_when_clicked(
     # AND: The attribution collapses on clicking it again
     attribution.click_attribution_link()
     assert not attribution.is_open
+
+
+@markers.test_case("C476304")
+@markers.parametrize("book_slug,page_slug", [("college-physics", "2-3-time-velocity-and-speed")])
+@markers.nondestructive
+def test_attribution_collapses_on_navigating_to_new_page(selenium, base_url, book_slug, page_slug):
+
+    # GIVEN: A page URL in the format of {base_url}/books/{book_slug}/pages/{page_slug}
+    # AND: The citation/attribution tab is open
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    attribution = content.attribution
+    attribution.click_attribution_link()
+
+    # WHEN: Navigating via next link
+    content.click_next_link()
+
+    # THEN: The citation/attribution section is not open on the new page
+    assert not attribution.is_open
+
+    attribution.click_attribution_link()
+
+    # WHEN: Navigating via Previous link
+    content.click_previous_link()
+
+    # THEN: The citation/attribution section is not open on the new page
+    assert not attribution.is_open
+
+    attribution.click_attribution_link()
```

Comment:
there is no assertion after this, so its not testing anything right? why is it here?

Reply:
my bad..removed it